opentelemetry-api/src/opentelemetry/propagators/composite.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+import typing
+
+from opentelemetry.context.context import Context
+from opentelemetry.trace.propagation import httptextformat
+
+logger = logging.getLogger(__name__)
+
+
+class CompositeHTTPPropagator(httptextformat.HTTPTextFormat):
+    """ CompositeHTTPPropagator provides a mechanism for combining multiple
+    propagators into a single one.
```

Comment:
It should include the list of arguments of `__init__`, otherwise the generated documentation will be missing it.

https://github.com/open-telemetry/opentelemetry-python/blob/005575e537ff97f345610c92276d6b55c32e65e5/opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py#L123-L125

Reply:
Good call, fixed